### ISSUE TITLE
Implement accessibility for friends only.

### ISSUE THREAD

**Issue by mihaimiron1** [nhcarrigan-spring-2026-cohort/amber-wisteria#53]
Accessibility/participation and visibility should be limited to friends only for meals
